nengo/tests/test_processes.py

```diff
@@ -557,9 +557,13 @@ def func(t):
             process = Piecewise({0.05: 0, 0.1: func}, interpolation="linear")
         assert process.interpolation == "zero"
 
+    def test_cubic_interpolation_warning(self):
+        pytest.importorskip("scipy")
+
         # cubic interpolation with 0 not in times
         process = Piecewise({0.001: 0, 0.1: 0.1}, interpolation="cubic")
         with pytest.warns(UserWarning, match="'cubic' interpolation.*for t=0.0"):
-            # this ValueError comes from inside Scipy
-            with pytest.raises(ValueError, match="number of derivatives at boundaries"):
+            try:
                 process.run(0.001)
+            except ValueError:
+                pass  # scipy may raise a ValueError
```

Comment:
Is scipy stochastically raising an error here? That seems really weird.

Reply:
Good question. When I first wrote this test, I didn't make it Scipy-specific. Then I realized we'll only get the warning if Scipy is installed, so I added the `importorskip`. Originally, I had a `pytest.raises` for this ValueError instead of the `try` block, and I'm not sure if I changed it because I wanted the test to run with or without Scipy, or if it was because TravisCI's Scipy did something different than mine. Either way, we don't really care about whether Scipy raises an error or not, so I think it's fine to leave it.